public-images/public-images-how-to/launch-qcow-with-qemu.rst

```diff
@@ -0,0 +1,129 @@
+.. _qcow-qemu:
+
+Launch QCOW Images Using QEMU
+=============================
+
+Ubuntu cloud images are released in many formats to enable many launch configurations and methods.
+
+Among the many formats for cloud images is the ``.img`` QCOW format. This article will cover the QCOW use case and provide
+instructions on how to use the images with QEMU.
+
+For starting with an ISO server image, see the `server documentation`_ on downloading and converting an ISO for use
+with QEMU.
+
+QEMU and QCOWs
+--------------
+
+`QEMU`_ is a system emulator and "type 2" virtual machine hypervisor. Type 2 hypervisors run on top of the host operating
+system. Often, QEMU is used with `KVM`_, which is a Type 1 hypervisor, which can run directly on bare metal. Together,
+QEMU and KVM can enable running virtual machines at near-native speeds.
+
+QCOW, which stands for "QEMU Copy On Write," is a format for images used by QEMU. QCOW image files grow according to use,
+as opposed to pre-allocating the entire image disk to a file. Ubuntu cloud images with the ``.img`` file type are QCOW
+images and are best used with the QEMU hypervisor.
+
+QEMU Shell
+~~~~~~~~~~
+
+When launching a system it is handy to know that ctrl-a c is an escape key that will take the user from the monitor or
+serial console and to the QEMU monitor. From this shell a user can send keys to the guest via the sendkey option or even
+close the guest via quit.
+
+Seeding the Image
+-----------------
+
+Since we are dealing with cloud images here, we will need a cloud-init datasource to use. See the guide in this series
+called `use-local-ds`_ for steps required for setting up a seed image for a local cloud-init datasource.
+
+Booting the Image with SeaBIOS
+------------------------------
+
+SeaBIOS booting an image is essentially the easiest as it is the default
+firmware used by QEMU.
+
+The following launches a cloud image with:
+
+* KVM acceleration
+* the local machine's CPUs
+* 2GB of memory
+* no graphics and serial output to the console
+* snapshot will make writes to a temporary file instead of the disk image
+  itself. This ensures the base disk is not touched. If at some point, you want
+  to persist the changes you've made on the disk, press C-a s
+* virtio network device that redirect's guest port 22 to host's port 2222
```

Comment:
```suggestion
* virtio network device that redirect's guest (VM) port 22 to localhost's port 2222 
```

Reply:
imo this suggestion reduces clarity. `localhost` is just inaccurate here.